monitoring/opencensus/noxfile_config.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2021 Google LLC
```

Comment:
Apologies, I should have mentioned this in my earlier review. This file only needs to be added if you want to change the default settings. It looks like the defaults are fine, so please delete this file.

Out of curiosity, were you going off of an existing sample when you created this directory or following the Authoring Guide? We're revamping our documentation so I'd love to know which docs that we're surfacing (or not surfacing). :)

Reply:
Deleted noxfile_config.py.

I was... kind of trying to do both? I used the Authoring Guide heavily and also ended up copying some stuff from existing samples, like things in the Dataflow directory.  This is my first contribution, and I'm not exactly a Python expert, so I likely made some obvious mistakes.

I think that a "how to send your first PR" or something like that would be helpful - almost like a codelab?  I think the Authoring Guide does a mostly good job, but it has a lot of guidelines that make the entire doc a bit unwieldy.  Someone like me could really use a dedicated guide on the mechanics, separate from the style guidelines.